crates/loans/src/benchmarking.rs

```diff
@@ -313,37 +337,87 @@ benchmarks! {
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(caller.clone()).into(), KBTC, deposit_amount));
     }: _(SystemOrigin::Signed(caller.clone()), KBTC)
     verify {
-        assert_last_event::<T>(Event::<T>::DepositCollateral(caller, KBTC, deposit_amount.into()).into());
+        assert_last_event::<T>(Event::<T>::DepositCollateral(caller, LEND_KBTC, expected_lend_tokens as u128).into());
     }
 
-    liquidate_borrow {
-        let alice: T::AccountId = account("Sample", 100, SEED);
-        let bob: T::AccountId = account("Sample", 101, SEED);
-        transfer_initial_balance::<T>(alice.clone());
-        transfer_initial_balance::<T>(bob.clone());
-        let deposit_amount: u32 = 200_000_000;
-        let borrowed_amount: u32 = 200_000_000;
-        let liquidate_amount: u32 = 100_000_000;
-        let incentive_amount: u32 = 110_000_000;
-        assert_ok!(Loans::<T>::add_market(SystemOrigin::Root.into(), LEND_DOT, pending_market_mock::<T>(KINT)));
-        assert_ok!(Loans::<T>::activate_market(SystemOrigin::Root.into(), LEND_DOT));
-        assert_ok!(Loans::<T>::add_market(SystemOrigin::Root.into(), KSM, pending_market_mock::<T>(LEND_KSM)));
-        assert_ok!(Loans::<T>::activate_market(SystemOrigin::Root.into(), KSM));
-        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(bob.clone()).into(), KSM, deposit_amount.into()));
-        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(alice.clone()).into(), LEND_DOT, deposit_amount.into()));
-        assert_ok!(Loans::<T>::deposit_all_collateral(SystemOrigin::Signed(alice.clone()).into(), LEND_DOT));
-        set_account_borrows::<T>(alice.clone(), KSM, borrowed_amount.into());
-    }: _(SystemOrigin::Signed(bob.clone()), alice.clone(), KSM, liquidate_amount.into(), LEND_DOT)
+    withdraw_all_collateral {
+        let caller: T::AccountId = whitelisted_caller();
+        transfer_initial_balance::<T>(caller.clone());
+        let deposit_amount: u128 = 200_000_000;
+        // divide by the default exchange rate
+        let rate: f64 = 0.02;
+        let expected_lend_tokens = deposit_amount as f64 / rate;
+        assert_ok!(Loans::<T>::add_market(SystemOrigin::Root.into(), KBTC, pending_market_mock::<T>(LEND_KBTC)));
+        assert_ok!(Loans::<T>::activate_market(SystemOrigin::Root.into(), KBTC));
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(caller.clone()).into(), KBTC, deposit_amount));
+        assert_ok!(Loans::<T>::deposit_all_collateral(SystemOrigin::Signed(caller.clone()).into(), KBTC));
+    }: _(SystemOrigin::Signed(caller.clone()), KBTC)
     verify {
-        assert_last_event::<T>(Event::<T>::LiquidatedBorrow(bob.clone(), alice.clone(), KSM, LEND_DOT, liquidate_amount.into(), incentive_amount.into()).into());
+        assert_last_event::<T>(Event::<T>::WithdrawCollateral(caller, LEND_KBTC, expected_lend_tokens as u128).into());
     }
 
+    // liquidate_borrow {
```

Comment:
Do we need these? If they're not working please add a comment why or remove

Reply:
Added a TODO comment to re-enable later